### PR TITLE
index.d entries for Eclipse Che container images

### PR DESCRIPTION
**Pull request by dharmit**
ping @kbsingh

### CODE REVIEW

File: index.d/codenvy.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  - id: 6
+    app-id: codenvy
+    job-id: centos_python34
+    git-url: https://github.com/dharmit/dockerfiles
+    git-branch: centos-dockerfiles
+    git-path: centos_python/3.4
+    target-file: Dockerfile
+    desired-tag: latest
+    notify-email: shahdharmit@gmail.com
```
Comment (bamachrn) [inline index.d/codenvy.yml]: @dharmit you missed depends-on for this entry?

Comment (dharmit) [inline index.d/codenvy.yml]: @bamachrn Done; seems like I yanked one line less. :wink:

**Comment by kbsingh** [comment]
Why the codenvy app-id. Should it not be eclipse ?

**Comment by kbsingh** [comment]
Lgtm

**Pull request merged**